sdk/synapse/synapse-spark/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "declarationDir": "./types",
+    "outDir": "./dist-esm",
+    "lib": ["dom"],
```

Comment:
we're supposed to avoid specifying `lib`: https://azure.github.io/azure-sdk/typescript_design.html#ts-config-lib

There are tricks to work around this, depending on what types you need from `dom`